OOModel/src/expressions/MetaCallExpression.cpp

```diff
@@ -62,4 +64,30 @@ MetaDefinition* MetaCallExpression::metaDefinition()
 	return ret;
 }
 
+Declaration* MetaCallExpression::generate()
+{
+	auto metaDef = metaDefinition();
+	if (!metaDef) return nullptr;
+
+	// every formal argument must have a value
+	if (arguments()->size() != metaDef->arguments()->size())
+	{
+		qDebug() << "#metaDefArgs != #metaCallArgs";
```

Comment:
Hmm, I should somehow come up with a way to report this type of errors to the user.